### PR TITLE
doc: process for adding a maintainer if the candidate does not have a role

### PR DESCRIPTION
**Pull request by jfischer-no**
There is no clear way for a candidate to assume the role of Maintainer if the candidate is neither a Contributor nor a Collaborator. To avoid disputes and confusion, candidates who are neither Contributors nor Collaborators must be approved by the TSC before they can assume the role of Maintainer.

### CODE REVIEW

**Review by fabiobaltieri — APPROVED** [review]

**Review by MaureenHelm — APPROVED** [review]

**Review by henrikbrixandersen — APPROVED** [review]
Thanks.

**Review by kartben — APPROVED** [review]

**Comment by keith-zephyr** [comment]
I added DNM label until the Process WG can discuss.  Since this proposes adding another responsibility to the TSC, we probably need to announce this to the TSC as well.

**Review by dkalowsk — APPROVED** [review]

**Review by tomi-font — COMMENTED** [review]
The way I understand it, the same is stated at the top of this page for candidates that are already collaborators:
> **Collaborator**: A highly involved Contributor in one or more areas.
**May become a Maintainer with approval of existing TSC voting members.**
https://github.com/zephyrproject-rtos/zephyr/pull/77933/files#diff-c557288edbce823dbb1409ebe4e78596230e4f869a9f61374bbf2bcf87c49bd8L18

Am I missing something? Could this be clarified?

**Comment by kartben** [comment]
> The way I understand it, the same is stated at the top of this page for candidates that are already collaborators:
> > **Collaborator**: A highly involved Contributor in one or more areas.
> **May become a Maintainer with approval of existing TSC voting members.**
> https://github.com/zephyrproject-rtos/zephyr/pull/77933/files#diff-c557288edbce823dbb1409ebe4e78596230e4f869a9f61374bbf2bcf87c49bd8L18
> 
> Am I missing something? Could this be clarified?

This basically says that the collaborator->maintainer path only requires one or more +1 from TSC members on the PR introducing the change.
The proposition in this PR is about requiring a formal vote from the TSC at large

**Comment by tomi-font** [comment]
> This basically says that the collaborator->maintainer path only requires one or more +1 from TSC members on the PR introducing the change.

Ah okay. I didn't quite understand this when reading the sentence I quoted.

**Comment by keith-zephyr** [comment]
This was discussed at the TSC and no objections raised.

**Review by keith-zephyr — APPROVED** [review]

**Review by nashif — APPROVED** [review]

**Pull request merged**
